enhancedTable/panel-manager.ts

```diff
@@ -316,9 +311,22 @@ export class PanelManager {
                 that.tableOptions.api.setFilterModel(newFilterModel);
             };
 
-            // determine if any column filters are present
+            // determine if any column filters are present and non static
+            // this determines if Clear Filters button is enabled or disabled
             this.anyFilters = function () {
-                return that.tableOptions.api.isAdvancedFilterPresent();
+                const columns = Object.keys(that.tableOptions.api.getFilterModel());
+                let noActiveColumns = true;
+
+                columns.forEach(column => {
+                    const columnConfigManager = new ColumnConfigManager(that.configManager, column);
+                    if (!columnConfigManager.isFilterStatic) {
+                        // if there is a non static column fiter, the clearFilters button is disabled
+                        noActiveColumns = false;
+                    }
+                });
+
+                // if column filters don't exist or are static, clearFilters button is disabled
+                return noActiveColumns;
```

Comment:
Can you change the function name and add a comment mentioning this returns `true` if there are *no* active filters